main.go

```diff
@@ -216,6 +219,11 @@ func main() {
 		MTLSconfig := mtls.NewMTLSConfig(mgr.GetClient(), operatorNamespace,
```

Comment:
I haven't spotted that before, but it `MTLSconfig` should start with a lower-case letter.